src/utility/index.jsx

```diff
@@ -608,10 +608,9 @@ export function fromNow(time) {
   return '';
 }
 
-export function displayHeroId(row, col, field, showGuide = false, imageSizeSuffix = IMAGESIZE_ENUM.SMALL.suffix, guideUrl, guideType) {
+export function displayHeroId(row, col, field, showGuide = false, guideUrl, guideType) {
```

Comment:
is this removing the sized images since we won't have access to them in the future?

Reply:
It's removed to prevent a linting error, since the image is no longer created here